torch/csrc/utils/python_arg_parser.h

```diff
@@ -769,13 +769,24 @@ static auto check_has_torch_function(PyObject* obj) -> bool
  *
  * 'torch_api' is a reference to a python torch API namespace.
  *
+ * 'torch_api_function' is the reference to the original torch method, usually,
+ * we can use torch_api and func_name to get torch_api_function. In some cases,
+ * e.g., torch custom op, we create the function in C++, if we still use
+ * torch_api and func_name to fetch original api, a cyclic call will happen.
+ *
+ * 'overloaded_args' is the args which have overloaded __torch_function__.
+ *
+ * 'func_name' is the named of the original torch method.
  */
 // Used for Tensor methods with arguments.
 auto handle_torch_function(PythonArgs &r, PyObject* self, PyObject* args, PyObject* kwargs, PyObject* torch_api, const char* module_name) -> PyObject*;
 
-// Used fpr functions.
+// Used for functions which needs to parse python args.
 auto handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyObject* torch_api, const char* module_name) -> PyObject*;
 
+// Used for functions created in C++, e.g., C++ custom op, whose args are already parsed.
```

Comment:
Technically, they're not already parsed; rather, we're doing overload handling prior to parsing, no?

Also, I feel it would be a lot clearer if we stop overloading the function here, and just gave separate names to each variant. There doesn't really seem to be a good reason to be overloading here.

Reply:
Sure, I will still keep handle_torch_function in this diff, and leave the renaming to the next diff. Otherwise, we will have quite a lot of distractive changes. There are quite a lot uses.

Reply:
Sure, just make the new function have a different name

Reply:
Sure, as you wish :-)